notebooks/tfx_pipelines/cicd/solutions/tfx_cicd_vertex.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
<p>Consider moving the line "Navigate to <a href="https://console.cloud.google.com/vertex-ai/pipelines" target="_blank">Vertex Pipelines</a> page in the Google Cloud Console." down to the end "Exercise: manually trigger CI/CD pipeline run with Cloud Build". This feels like we tell them to go to Vertex Pipelines without a specific action / purpose when they need to switch back to continue. Instead, at the end of this exercise, they will instead see the Cloud Build triggered pipeline run to check out.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done!</p>